packages/genesys-spark-components/src/components/stable/gux-popup/gux-popup.tsx

```diff
@@ -33,6 +34,12 @@ export class GuxPopup {
   private popupElementContainer: HTMLElement;
   private cleanupUpdatePosition: ReturnType<typeof autoUpdate>;
 
+  /**
+   * Placement of the popup. Default is bottom-start
+   */
+  @Prop({ mutable: true })
```

Comment:
I don't think setting `mutable: true` here is correct as you are not changing the value of the property from inside the component. Look at the Stencil Docs https://stenciljs.com/docs/properties#prop-mutability-mutable

Reply:
I was thinking this also but then thought isnt the `placement` property dynamically updated within the component when the position changes. I am not sure does a dynamic update to a property within the component mean its mutable or not 🤔

Reply:
@daragh-king-genesys good catch. I had copied the `placement` prop declaration from `gux-tooltip` and notice it's set as `mutable: true` in `gux-tooltip` as well. It doesn't look like either component is setting a new value of the property from inside the component like you mentioned. Mind if I also remove `mutable: true` for `gux-tooltip`?

Reply:
@gavin-everett-genesys is this where you're referring to `placement` being dynamically changed inside `gux-popup`?

```
void computePosition(this.forElement, this.root, {
      placement: this.placement,
      strategy: 'fixed',
      middleware: middleware
    }).then(({ x, y, middlewareData }) => {
...
```

^^ I don't think `this.placement` is set a new property value, but correct me if I'm wrong here.